doc/help/kdb-set.md

```diff
@@ -30,7 +30,9 @@ To set a key to a negative value, `--` has to be used to stop option processing.
 - `-C`, `--color <when>`:
   Print never/auto(default)/always colored output.
 - `-v`, `--verbose`:
-  Explain what is happening.
+  Explain what is happening. Also shows Configfile and Mountpoint in case of an error/warning
+- `-d`, `--debug`:
```

Comment:
You do not need to document this for every command individually. Please document it once in doc/help/kdb.md

Please also make sure that the behavior when errors/warnings are shown is identical for every program (with tests).

Reply:
> Please also make sure that the behavior when errors/warnings are shown is identical for every program (with tests).

What is "every program"?
Also which kind of tests do you think of? shell recorder?

Reply:
> What is "every program"?

Web-UI you already showed. (Thank you, looks very nice) Then also manual tests for qt-gui. And then for `kdb` suite (complete convert cp editor export file find fstab get globalmount import ls metaget metals metaremove metaset mount mv remount rm set sget (should never print an error) shell specmount umount validation). For merge tools you do not need to make tests, as they are reworked now anyway.

> Also which kind of tests do you think of? shell recorder?

For kdb tools shell recorder. web-ui and qt-gui manual tests are enough, as automation of these tests would be too much effort.

Reply:
Snippet-conversion (our web-page) a manual test is also enough.

Reply:
Ok, do you know an easy way how to trigger an error/warning for each of these commands? Or is this case dependent?

Reply:
With the trigger/error (error plugin) you should be always able to trigger an error (even when only getting config). Maybe you can write a single loop to test most of the programs but then you cannot use shell recorder. I think its better if you start with simple individual shell-recorder tests and then progress to a loop similar to tests/shell/check_import.sh but iterating over commands and not plugins.

Btw. please bring also tests/shell/check_error.sh up-to-date.